src/components/core/pivotTable/tests/agGridApiWrapper.spec.ts

```diff
@@ -0,0 +1,32 @@
+// (C) 2007-2019 GoodData Corporation
+import ApiWrapper from '../agGridApiWrapper';
+
+describe('agGridApiWrapper', () => {
+    describe('getHeaderHeight()', () => {
+        it('should return height of grid header', () => {
+            const mockApi = {
```

Comment:
I think we should not mock agGrid wrapper. Tests should fail when agGrid api changes with version increase.

Reply:
Please take a look at my solution how to test API wrapper functions on real ag-grid API object.